### ISSUE TITLE
Steps to download precomputed QR->A logits

### ISSUE THREAD

**Issue by hemanths03** [iLearn-Lab/TIP23-ARC#1]
Hello,

I am unable to download pre computed QR->A logits and representations. Also please provide the checkpoints and kindly guide me to replicate the results of the paper.

**Comment by SDLZY** [comment]
Hi,
 I have uploaded the pre computed QR->A logits and representations. Thank you for your interest in our work.
